[OSDOCS#17176]: Make changes as per the errors and warnings for CQA-1 (#114179)
openshift/openshift-docs: modules/security-deploy-signature.adoc

[id="security-deploy-signature_{context}"]
= Using signature transports

[role="_abstract"]

> A signature transport is a way to store and retrieve the binary signature blob. There are two types of signature transports.

Reviewer: ```suggestion
You can use a signature transport as a way to store and retrieve the binary signature blob. 

Signature transports can be either of the following types:
```
